models/user.coffee

```diff
@@ -22,6 +22,14 @@ module.exports = class User extends Model
   isPremium: ->
     @get('is_premium')
 
+  homePath: ->
+    if parseInt(@get('following_count')) <= 1 then '/explore' else (@get('home_path') or '/')
+
+  hasCustomHomePath: ->
+    @get('home_path') != '/explore' and
+    @get('home_path') != '/' and
+    @get('home_path') != "/#{@get('slug')}"
```

Comment:
https://github.com/aredotna/ervell/pull/723/commits/1ea2f2d1c7258195848e75a68d425610b92a22cc#diff-0249de556acb4d3659af85ff67643708R2

also it's not used anyway. gonna delete it